packages/@aws-cdk/aws-lambda/lib/code.ts

```diff
@@ -160,6 +162,54 @@ export class InlineCode extends Code {
   }
 }
 
+/**
+ * Bundle options
+ */
+export interface BundleOptions {
+  /**
+   * The Docker image where the command will run.
+   */
+  readonly image: DockerImage;
+
+  /**
+   * The command to run in the container.
+   */
+  readonly command: string[];
+
+  /**
+   * Docker volumes to mount.
+   *
+   * @default - The path to the asset file or directory is mounted at /asset
```

Comment:
Wondering if we should *always* mount the source and this will just be additional volumes. I can't see a reason why not...